components/create-bounty-modal.tsx

```diff
@@ -611,13 +611,15 @@ export default function CreateBountyModal() {
   }, [customTokens]);
 
   useEffect(() => {
+    let approved = true
```

Comment:
It should be `false` by default, this way we prevent the user from getting the `exceeds allowance` error if the app fails to update the allowance

Reply:
yeah, i thought the same, but when the type is foundingBounty the approval must be true, therefore i will improve that.

Reply:
I'd add that explanation as a comment and I think it does suffice